src/components/FirefoxReleaseVersionMarkers.svelte

```diff
@@ -1,33 +1,29 @@
 <script>
-import { getContext } from 'svelte';
-import Marker from 'udgl/data-graphics/guides/Marker.svelte';
-import { store } from '../state/store';
-import { firefoxVersionMarkers } from '../state/product-versions';
+  import { getContext } from 'svelte';
+  import Marker from 'udgl/data-graphics/guides/Marker.svelte';
+  import { store } from '../state/store';
+  import { firefoxVersionMarkers } from '../state/product-versions';
 
+  export let labels = true;
 
-export let labels = true;
-
-const xScale = getContext('xScale');
-
-let markers = [];
-firefoxVersionMarkers.subscribe((m) => { markers = m; });
+  const xScale = getContext('xScale');
 
+  let markers = [];
+  firefoxVersionMarkers.subscribe((m) => {
+    markers = m;
+  });
 </script>
 
-
-<g class='firefox-release-version-markers'>
+<g class="firefox-release-version-markers">
   {#if markers && markers.length}
-    {#each markers
-      .filter((d) => d.date !== undefined && d.date >= $xScale.domain()[0] && d.date <= $xScale.domain()[1]) as {label, date}, i (date)}
+    {#each markers.filter((d) => d.date !== undefined && d.date >= $xScale.domain()[0] && d.date <= $xScale.domain()[1]) as { label, date }, i (date)}
       <Marker location={date}>
         {#if labels}
           {#if $store.productDimensions.channel === 'nightly'}
             {label + 2}
           {:else if $store.productDimensions.channel === 'beta'}
             {label + 1}`
-          {:else}
-            {label}
-          {/if}
+          {:else}{label}{/if}
```

Comment:
This rule is weird. I definitely prefer it the way it was.

Reply:
Yeah that is weird. Looking at the prettier-plugin-svelte docs, I don't know if this can be changed. I bet it'll be fixed in a future release, though.

Reply:
It still fixes more than it makes things weird. r+ if everything seems to work locally